recipe/meta.yaml

```diff
@@ -12,16 +12,18 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<37]
```

Comment:
Just wondering why this is necessary - [upstream suggests that it works for Python 3.5](https://github.com/microsoft/onnxconverter-common/blob/91ef4b376f3186267067952c0150516a3c148f4e/setup.py#L53)

Reply:
I've updated the pinning to match upstream.